runtime/onert/core/src/exec/ExecTime.h

```diff
@@ -94,7 +94,7 @@ class ExecTime
   /**
    * @brief Update metrics file with new data.
    */
-  void uploadOperationsExecTime() const { _json.uploadOperationsExecTime(); }
+  void storeOperationsExecTime() const { _json.storeOperationsExecTime(); }
```

Comment:
`upload` gave me some thoughts that `upload to somewhere?` but actually it saves to `_measurement_file` file. we can also choose `save` or `update` if suits better

Reply:
I agree.